packages/delete/src/utils/deleteRequestParser.js

```diff
@@ -0,0 +1,77 @@
+import _get from 'lodash/get'
+import PropTypes from 'prop-types'
+
+const deleteStatus = {
+  DELETABLE: 'DELETABLE',
+  DENIED_BY_CASCADE: 'DENIED_BY_CASCADE',
+  NO_DELETE_PERMISSION: 'NO_DELETE_PERMISSION'
+}
+
+const transformRelatedEntities = (relatedEntities, entityToDelete, deletable, currentBuId) => ({
```

Comment:
🙈  is this tested well enough? :)

Reply:
i think so. I started writing many tests but ended up copy alot of test data and do duplicate stuff. So i decided to pack all special cases in this one test and compare the whole output.